### ISSUE TITLE
[IMPROVEMENT] Code markup (codestyle)

### ISSUE THREAD

**Issue by soundstorm** [RocketChat/Rocket.Chat.iOS#1245]
When using three backticks the first row (language style) is shown as plain text (if there is any) with trailing gray background.

e.g.

swift`                  `

Or if using no language definition we are showing a blank code line.

**Comment by rafaelks** [comment]
You mean, if you do something like this... ?

![screen shot 2018-02-15 at 09 01 01](https://user-images.githubusercontent.com/551004/36253462-c3b07f0a-122e-11e8-9af4-4a2ebfe41b72.png)

**Comment by soundstorm** [comment]
@rafaelks yep. This is displayed as plain "swift" with a trailing code block-line.

**Comment by soundstorm** [comment]
Now some new behaviours:

If you write text before the last backticks, they'll be shown as independent code block:

```js
document.getElementById('me');
```
`end.`

which is
```
'''
document.getElementById('me');
end.'''
```

When you insert something like
```
'''
asdf'''
```
you'll get
```js
```
`end.`
